tutorial/index.jade

```diff
@@ -5,99 +5,56 @@
     :markdown
-      ## Registration server side implementation
-
-      It's now the controller's responsibility to verify that the user provides the required information to
-      create a new user. If all goes good the controller should persist it in the database, registers the user under
-      Authy's API and requests the send of the verification code to the user through the `RequestPhoneNumberConfirmationTokenAsync`
-      method invocation, which will be explained a few sections ahead.
+      ## New User Form
 
-      The flow finalizes with the redirect of the user to the verification code page.
+      When we create a new user, we ask for a name, e-mail address, and a password.
+      In order to validate their account we also ask them for a mobile number
+      with a country code. We use Authy to send them a one-time password via SMS
+      to this phone number.
 
-  .step(data-title='Verification form and UI' data-file='AccountVerification.Web/Views/Account/VerifyRegistrationCode.cshtml'
-        data-mode='html')
-    :markdown
-      ## Verification form and UI
+      It is now the controller's responsibility to verify that the user provides
+      the necessary information to create a new user. If the user is created successfully,
+      he/she will be logged into the system automatically.
 
-      As stated before, once the user completes the registration they get redirected to the verification page, which you may access
-      at any time using the route `/Account/VerifyRegistrationCode`. This page consists of a simple form where you can input your
-      account E-mail address, and verify the SMS verification code.
+      Now the user is logged in but not verified. In the next steps we'll
+      learn how to verify the user using Authy.
 
-  .step(data-title='Verification server side implementation' data-file='AccountVerification.Web/Controllers/AccountController.cs'
-        data-highlight='73-97')
+  .step(data-title='Authy configuration' data-file='AccountVerification.Web/Web.config' data-highlight="19-22" data-mode='xml')
     :markdown
-      ## Verification server side implementation
-
-      The server side implementation for the registration process confirmation ensures the given E-mail belongs to a previously registered user which hasn't
-      performed the verification process yet.
-
-      From this moment the process has two critical steps:
-      1. It communicates with Authy's API to check that the given code is correct
-
-      1. Once the code is verified with Authy we send a confirmation SMS to the user.
-
-      1. Finally it redirects the user to a success page.
-
-  .step(data-title='Twilio and Authy configuration' data-file='AccountVerification.Web/Web.config' data-highlight="19-22" data-mode='xml')
-    :markdown
-      ## Twilio and Authy configuration
+      ## Authy configuration
```

Comment:
In this step, in the line 62 you can specify `xml` as the language for this code snippet.  Something like:

```
```xml
   ... Your snippet
```